ui/src/components/LoginForm/SessionTable.jsx

```diff
@@ -1,144 +1,134 @@
-export default function SessionTable(props) {
   return (
     <Table bordered hover size="sm" responsive>
       <thead>
         <tr>
-          {props.params.showBeamline && <th>Beamline</th>}
-          <th>Title</th>
-          <th>Start</th>
-          <th>End</th>
-          <th>Portal</th>
-          <th>User</th>
-          <th>Logbook</th>
+          <th scope="col">ID</th>
+          {showBeamline && <th scope="col">Beamline</th>}
+          <th scope="col">Title</th>
+          <th scope="col">Start</th>
+          <th scope="col">End</th>
+          <th scope="col">Portal</th>
+          <th scope="col">User</th>
+          <th scope="col">Logbook</th>
         </tr>
       </thead>
       <tbody>
-        {props.sessions.map((session) => {
-          const variant =
-            props.selectedSessionId === session.session_id
-              ? styles.selected_row
-              : styles.unselected_row;
-
-          return (
-            <tr
-              key={session.session_id}
-              onClick={() => {
-                props.onSessionSelected(session);
-              }}
-              className={variant}
-            >
-              <td>{getProposalBySession(session)}</td>
-              {props.params.showBeamline && <td>{session.beamline_name}</td>}
-              <td>{session.title}</td>
-              <td>
-                {getScheduledDateComponent(
-                  session,
-                  session.start_date,
-                  session.start_time,
-                )}
-                <br />
-                {session.is_rescheduled &&
-                  getDateComponent(
-                    session.actual_start_date,
-                    session.actual_start_time,
-                  )}
-              </td>
-              <td>
-                {getScheduledDateComponent(
-                  session,
-                  session.end_date,
-                  session.end_time,
-                )}
-                <br />
-                {session.is_rescheduled &&
-                  getDateComponent(
-                    session.actual_end_date,
-                    session.actual_end_time,
-                  )}
-              </td>
-              <td>{getLinkBySession(session)[0]}</td>
-              <td>{getLinkBySession(session)[1]}</td>
-              <td>{getLinkBySession(session)[2]}</td>
-            </tr>
-          );
-        })}
-      </tbody>{' '}
+        {sessions.map((session) => (
+          <tr
+            key={session.session_id}
+            className={styles.row}
+            data-selected={selectedSessionId === session.session_id}
+            tabIndex={0}
+            onClick={() => onSessionSelected(session)}
+            onKeyDown={(evt) => {
+              // For keyboard accessibility; ideally add "Select" button to each row instead of making rows clickable
+              if (evt.key === 'Enter' || evt.key === ' ') {
+                onSessionSelected(session);
+              }
+            }}
```

Comment:
I'm including a bunch of accessibility fixes like this one and various `aria-label` attributes.

Reply:
Very nice !